src/components/Header/index.tsx

```diff
@@ -142,13 +143,14 @@ const MobileBalanceText = styled(Text)`
   font-weight: 500;
 `
 
-export const NETWORK_LABELS: { [chainId in ChainId]: string | null } = {
+export const NETWORK_LABELS: any = {
   [ChainId.MAINNET]: 'Ethereum',
   [ChainId.RINKEBY]: 'Rinkeby',
   [ChainId.ROPSTEN]: 'Ropsten',
   [ChainId.GÖRLI]: 'Görli',
   [ChainId.KOVAN]: 'Kovan',
-  [ChainId.FUSE]: 'Fuse'
+  [ChainId.FUSE]: 'Fuse',
```

Comment:
from where the `ChainId` come from? is it possible to add binance there too?
Also there should be difference between BSC and BSC testnet here. theoretically at least, right?

Reply:
we use `BINANCE_CHAIN_ID` to separate bsc and bsc testnet, in qa it uses the testnet chainid and in production it uses the production chainid